src/ExpressionEvaluator/Core/Source/ExpressionCompiler/DkmUtilities.cs

```diff
@@ -155,7 +155,7 @@ internal static object GetSymReader(this DkmClrModuleInstance clrModule)
                 Access: resultProperties.AccessType,
                 StorageType: resultProperties.StorageType,
                 TypeModifierFlags: resultProperties.ModifierFlags,
-                CustomTypeInfo: null);
+                DeclaredTypeInfo: null);
```

Comment:
Oh, I would have left this one as "CustomTypeInfo" since there isn't a corresponding "DeclaredType".